frontend/sentry.client.config.ts

```diff
@@ -1,12 +1,10 @@
-// This file configures the initialization of Sentry on the browser.
-// The config you add here will be used whenever a page is visited.
-// https://docs.sentry.io/platforms/javascript/guides/nextjs/
-
+import { VERCEL_ENV } from '@config/env';
 import { isCurrentProductionDeployment } from '@helpers/vercel-helpers';
 import * as Sentry from '@sentry/nextjs';
 import { BrowserTracing } from '@sentry/browser';
 
 const SENTRY_DSN = process.env.NEXT_PUBLIC_SENTRY_DSN;
+const sampleRate = VERCEL_ENV === 'production' ? 0.05 : 0;
```

Comment:
As far as I remember, *all* our vercel deployments are "production", so this includes all pull request code.

Reply:
The NODE_ENV is production, but not the VERCEL_ENV

![image](https://github.com/iFixit/react-commerce/assets/52104630/18114acf-e9ab-4d82-b74a-6ce6f3316aa6)